lib/diagnostic.rb

```diff
@@ -78,10 +94,14 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat and I go meow"
 
 ## Explain why this would be the output, based on the method lookup chain.
-# your response as a comment here
+
+# I'm not very comfortable with self yet to know exactly why but I think it has to do
+# with the fact that in ruby its going to look up at the say_hello method that HouseCat inherits from Cat
```

Comment:
`say_hello` isn't inherited from `Cat`, but from `Animal`, which is a superclass of `Cat`, which is a superclass of `HouseCat`.